blog/2024-04-22-feature-observability-semantic-conventions.md

```diff
@@ -0,0 +1,46 @@
+---
+slug: "feature-observability-semantic-conventions"
+title: "Feature Observability Semantic Conventions"
+date: 2024-04-22
+authors: [dyladan]
+description: "Help us make feature observability better for everyone!"
+tags: [otel, spec, specification, opentelemetry, observability]
+draft: false
+---
+
+As comic book aficionados say, with great power comes great responsibility, or great complexity as the case may be.
+The OpenTelemetry feature observability semantic convention SIG aims to combat this problem by defining a consistent set of rules and definitions for feature telemetry data.
```

Comment:
```suggestion
The OpenTelemetry feature observability semantic convention special interest group (SIG) aims to combat this problem by defining a consistent set of rules and definitions for feature telemetry data.
```

Could you please link to the charter of the semantic convention SIG?

Reply:
"feature observability semantic convention special interest group" is the SIG and this document is its proposed charter

Reply:
We can link to the PR for the charter but I can't link directly to the charter file in the repo until it is merged. There isn't one for general semconv https://github.com/open-telemetry/community/tree/main/projects